src/cljs/proton/core.cljs

```diff
@@ -108,9 +109,7 @@
 
           ;; wipe existing config
           (atom-env/insert-process-step! "Wiping existing configuration")
-          (doall (map atom-env/unset-config! (filter #(not
-                                                        (or (= "core.disabledPackages" %)
-                                                            (= "core.themes" %))) (atom-env/get-all-settings))))
+          (doall (map atom-env/unset-config! (filter #(not (= "core.themes" %)) (atom-env/get-all-settings))))
```

Comment:
We had a faulty state when packages were disabled / uninstalled. So even if a package gets uninstalled and installed again, it is still deactivated and proton currently has no way of activating one. 

This conflicted with the uninstallation process of, for example, linters. Apparently the atom API doesn't return deactivated packages as `installed` (at least in my tests) so we couldn't purge them. They would just remain as disabled zombie packages.

I tested this a bit and it seems like the problem we initially tried to solve with this (packages getting activated right away after they are installed through proton) is not present anymore. Though we need to test this a bit more :)